ConsoleApp/Program.cs

```diff
@@ -0,0 +1,275 @@
+        ReadKey();
+    }
+
+
+
+}
+
+void UpdateContact()
+{
+    Clear();
+
+    WriteLine("Enter name to find contact:");
+    var nametoupdate = Console.ReadLine();
+
+    WriteLine("Enter surname to find contact:");
+    var lastnametoupdate = Console.ReadLine();
+    try
+    {
+        var contacts = ReadContacts(filename);
+        var newContacts = new (string, string, string)[contacts.Length];
+        Array.Copy(contacts, newContacts, contacts.Length);
+
+        var index = 0;
+
+        WriteLine("Enter new first name:");
+        var firstName = Console.ReadLine();
+
+        WriteLine("Enter new last name:");
+        var lastName = Console.ReadLine();
+
+        WriteLine("Enter new phone:");
+
+        var newphone = Console.ReadLine();
+        while (index < newContacts.Length)
+        {
+            if (newContacts[index].Item1 == nametoupdate && newContacts[index].Item2 == lastnametoupdate)
+            {
+
+
+                newContacts[index].Item1 = firstName;
+                newContacts[index].Item2 = lastName;
+                newContacts[index].Item3 = newphone;
+                break;
+            }
+
+            index++;
+
+        }
+
+        var csvBuilder = new StringBuilder();
+        foreach (var contact in newContacts)
+        {
+            csvBuilder.AppendLine(Serialize(contact));
+        }
+
+        File.WriteAllText(filename, csvBuilder.ToString());
+
+        if (contacts[index] != newContacts[index])
+        {
+            WriteLine("Contact renamed, press any key to continue");
+        }
+        ReadKey();
+    }
+    catch (FileNotFoundException Nofile)
+    {
+
+        Console.WriteLine($"Phonebook file is not found, please add a new contact or put the phonebook file.csv in the correct directory");
+        ReadKey();
+    }
+
+}
+
+void RemoveContact()
+{
+    Clear();
+
+    WriteLine("Enter phone to remove:");
+    var phoneToRemove = Console.ReadLine();
+    try
+    {
+        var contacts = ReadContacts(filename);
+        var newContacts = new (string, string, string)[contacts.Length];
+        Array.Copy(contacts, newContacts, contacts.Length);
+
+        var index = 0;
+
+        while (index < newContacts.Length)
+        {
+            if (newContacts[index].Item3 == phoneToRemove)
+            {
+                newContacts[index] = newContacts[^1];
+                Array.Resize(ref newContacts, newContacts.Length - 1);
+            }
+            else
+            {
+                ++index;
+            }
+        }
+
+
+        var csvBuilder = new StringBuilder();
+        foreach (var contact in newContacts)
+        {
+            csvBuilder.AppendLine(Serialize(contact));
+        }
+
+        File.WriteAllText(filename, csvBuilder.ToString());
+
+        WriteLine($"{contacts.Length - newContacts.Length} Contact(s) removed, press any key to continue");
+        ReadKey();
+    }
+    catch (FileNotFoundException Nofile)
```

Comment:
it should be
```suggestion
    catch (FileNotFoundException noFile)
```

but better name it
```suggestion
    catch (FileNotFoundException fileNotFoundException)
```

or
```suggestion
    catch (FileNotFoundException fnfe)
```